capacities_test.go

```diff
@@ -0,0 +1,57 @@
+package packngo
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestAccCheckCapacity(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+	c := setup(t)
+
+	input := &CapacityInput{
+		[]ServerInfo{
+			{
+				Facility: "ams1",
+				Plan:     "baremetal_0",
+				Quantity: 1},
+		},
+	}
+
+	cap, _, err := c.CapacityService.Check(input)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, s := range cap.Servers {
+		if !s.Available {
+			t.Fatal(fmt.Errorf("Capacity of %d severs should have been available.", input.Servers[0].Quantity))
+			break
+		}
+	}
+
+	list, _, err := c.CapacityService.List()
+	if err != nil {
+		t.Fatal("List of capacities not fetched")
+	}
+
+	for k, v := range *list {
+		if v["baremetal_2a2"].Level == "unavailable" {
+			input.Servers[0].Plan = v["baremetal_2a2"].Level
```

Comment:
This is wrong - you are copying string `"unavailable"` to the Plan attr. Then, the subsequent c.CapacityService.Check sends request with body.
```
{"servers":[{"facility":"sjc1","plan":"unavailable","quantity":1}]}
```

It throws an error, but it's not because some plan is not available, it's because (verbatim from the reply) `{"errors":["Plan unavailable is invalid"]}`